tuxemon/core/item.py

```diff
@@ -172,8 +201,14 @@ class with the name of the effect you define in item.db.
 
         # Loop through all the effects of this technique and execute the effect's function.
         for effect in self.effect:
-            last_effect_name = str(effect)
-            result = getattr(self, last_effect_name)(user, target)
+            effect_lines = str(effect).split()
+            effect_name = effect_lines[0]
+            effect_param = effect_lines[1] if len(effect_lines) > 1 else None
+
+            if effect_param:
```

Comment:
Please add a `TODO`; something like `TODO: refactor effects to remove the need of getattr`

Reply:
Just going to actually do the thing, rather than leave a crappy implementation.